content/docs/Getting started/deploying-rqd.md

```diff
@@ -89,6 +96,12 @@ docker build -t opencue/rqd -f rqd/Dockerfile .
 docker run -td --name rqd01 --env CUEBOT_HOSTNAME=${CUEBOT_HOSTNAME} --volume "${CUE_FS_ROOT}:${CUE_FS_ROOT}" opencue/rqd
 ```
 
+-   **In both Option 1 and 2**, if running the RQD container in your local development machine, use the `--add-host` flag on the `docker run` command as follows:
+
+    ```shell
+    docker run -td --name rqd01 --env CUEBOT_HOSTNAME=${CUEBOT_HOSTNAME} --volume "${CUE_FS_ROOT}:${CUE_FS_ROOT}" --add-host host.docker.internal:host-gateway opencue/rqd
```

Comment:
> Is there a downside to just adding the --add-host flag into the example commands for Options 1 and 2? Does it cause problems if it's not actually needed?

Yes, that's a good point. We can have the `--add-host` flag but it would need to be mapped with the Cuebot host IP.

Shall I remove the edge case command that I'd just added and instead, include the flag in the existing option 1 and 2 commands and generalize it as `--add-host <CUEBOT HOST IP MAPPING>` with a link to the `docker run` [documentation](https://docs.docker.com/engine/reference/commandline/run/#add-host)?

Reply:
> --add-host flag but it would need to be mapped with the Cuebot host IP

Not sure I follow. Why do we need to do any additional mapping?

I'm suggesting to always use `--add-host host.docker.internal:host-gateway` in the command.

* If Cuebot is running on the host machine, `CUEBOT_HOSTNAME` will be `host.docker.internal`, and `--add-host host.docker.internal:host-gateway` will allow RQD to communicate with Cuebot.
* If Cuebot is running in a Docker container, `CUEBOT_HOSTNAME` will be the container IP, and `--add-host host.docker.internal:host-gateway` will do nothing.
* If Cuebot is running on some other machine, `CUEBOT_HOSTNAME` will be the hostname/IP of that machine, and `--add-host host.docker.internal:host-gateway` will do nothing.

I'm assuming here that `--add-host host.docker.internal:host-gateway` will not cause problems in the second and third cases when it's not actually needed. I haven't tested this to confirm that.

Reply:
I see now, didn't quite think of it that way. 😄 
Yes, since the flag has no influence on the `CUEBOT_HOSTNAME` parameter (which is changed depending on where Cuebot is run) it should in theory have no effect in cases 2 and 3.

> I haven't tested this to confirm that.

I could set up Cuebot in a VirtualBox VM with a bridged network adapter and test it out, just to be certain.

Reply:
> I could set up Cuebot in a VirtualBox VM with a bridged network adapter and test it out, just to be certain.

You were right, tested and confirmed that  `--add-host host.docker.internal:host-gateway` does not have an impact.
Tested with RQD running in a VM:
![rqd-vm-screenshot](https://github.com/AcademySoftwareFoundation/opencue.io/assets/24616975/40bf9fd5-e1cd-4aec-b337-4b738010f64a)

Doc updated in https://github.com/AcademySoftwareFoundation/opencue.io/pull/282/commits/47ea3551341b414a370de015cfa297f201c99286